docs/Makefile

```diff
@@ -1,14 +1,31 @@
 default: install
 
+build_dir=build
+dest_dir=$(build_dir)/$(shell sed -En "s/^baseurl: \"(.*)\"/\1/gp" _config.yaml)
+
+.PHONY: help
 help:
 	@egrep '(^\S)|^$$' Makefile
 
+.PHONY: install
 install:
 	bundle config set --local path vendor/bundle
 	bundle install
 
+.PHONY: upgrade
 upgrade:
 	bundle update
 
-s serve:
-	bundle exec jekyll serve --source sample_site --destination build/ --livereload --trace
+.PHONY: build
+build:
+	rm -rf $(dest_dir)
+	bundle exec jekyll build --trace --source . --destination $(dest_dir)
+
+.PHONY: check
```

Comment:
bad phony

Reply:
the name of the Makefile target is `check` I think .PHONY is right.

Reply:
I'm sorry. I'm blind.